alacritty_macos.yml

```diff
@@ -315,6 +315,8 @@ key_bindings:
   - { key: Key0,     mods: Command, action: ResetFontSize                }
   - { key: Equals,   mods: Command, action: IncreaseFontSize             }
   - { key: Minus,    mods: Command, action: DecreaseFontSize             }
+  - { key: K,        mods: Command, action: ClearHistory                 }
+  - { key: K,        mods: Command, chars: "\x0c"                        }
```

Comment:
Oh nice, I didn't know you could bind multiple actions the same key binding. This solves the issue I was having more easily.

Reply:
You couldn't. Until I wrote this patch to make it possible. :D 

Check out the changes in `src/input.rs`, it was fairly simple so I thought that this would be the best solution.

Reply:
Yea I can see that. It's not immediately obvious, but makes sense why it works. I can imagine some weird binding shadowing issues but generally I like this.

I mostly like it here because it keeps the clear history action straightforward, without needing to worry about moving the lines.

Reply:
With the previous configuration double bindings would just silently be ignored, like this they're both used. If it would have caused errors previously I think this would be a far more severe change, but like this it should actually make things more transparent and less error prone.

Or that was my train of thought at least. But I was trying to justify the change after the fact. :D

Reply:
I'd love to see someone on macOS actually verify this, but it worked just fine when I changed command to control and added it to my keybindings.

Reply:
Yeah completely agree with that sentiment. I tested it on Linux by changing it to `Control`, but it's always better to be safe than sorry. :D

Reply:
Maybe @saulshanabrook can take a look?